### PR TITLE
Drastically limit wait_for_event_queue_length for pytest runs

### PR DESCRIPTION
**Pull request by inducer**
cc @nchristensen

### CODE REVIEW

File: meshmode/array_context.py [hunk 1]
```diff
@@ -264,37 +264,37 @@ class PyOpenCLArrayContext(ArrayContext):
...
         """
         super().__init__()
         self.context = queue.context
         self.queue = queue
         self.allocator = allocator if allocator else None
 
-        if wait_event_queue_legnth is not None:
-            wait_event_queue_legnth = 10
+        if wait_event_queue_length is not None:
+            wait_event_queue_length = 10
```
Comment (alexfikl) [inline meshmode/array_context.py:295]: Is this intended? It seems like the length is either `None` or `10`?

Comment (inducer) [reply meshmode/array_context.py:295]: D'oh! Lack of sleep?

Comment (inducer) [reply meshmode/array_context.py:295]: 08c2e09

Comment (inducer) [reply meshmode/array_context.py:295]: Also, thanks for catching that!

**Pull request merged**